specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/stable/2019-04-01/containerregistry_build.json

```diff
@@ -793,6 +799,10 @@
           "$ref": "#/definitions/SourceTriggerDescriptor",
           "description": "The source trigger that caused the run."
         },
+        "timerTrigger": {
```

Comment:
Could we add new properties at the end of `"properties": { … }` ?

Reply:
Sure. See update